src/reference-manual/mcmc.Rmd

```diff
@@ -50,8 +50,8 @@ details of the geometry.
 
 In Stan, this matrix may be set to the identity matrix (i.e., unit
```

Comment:
How about we replace this whole paragraph with "In Stan, this matrix is usually estimated from warmup draws, though it can be set manually. By default only the diagonal is estimated."

I think the goal of that paragraph is to just give a vague idea of what Stan does. More details can be left to the other bits of doc.